src/test/java/com/study/racing/SetCollectionTest.java

```diff
@@ -0,0 +1,53 @@
+package com.study.racing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetCollectionTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@Test
+	@DisplayName("요구사항 1")
+	public void case_1() {
+		assertThat(numbers).size().isEqualTo(numbers.size());
+	}
```

Comment:
다음과 같이도 작성해볼 수 있을 것 같아요. 다양한 API를 활용해보시는 것도 좋을것 같아요~.
```suggestion
 assertThat(numbers).hasSize(3);
```